plugins/woocommerce/includes/admin/helper/class-wc-helper-admin.php

```diff
@@ -27,7 +27,14 @@ class WC_Helper_Admin {
 	 * @return void
 	 */
 	public static function load() {
-		add_filter( 'woocommerce_admin_shared_settings', array( __CLASS__, 'add_marketplace_settings' ) );
+		global $pagenow;
+
+		$is_in_app_marketplace = ( 'admin.php' === $pagenow && ( isset( $_GET['page'] ) && 'wc-admin' === $_GET['page'] ) && ( isset( $_GET['path'] ) && '/extensions' === $_GET['path'] ) );
```

Comment:
Nitpick: Since these are only && can't we get rid of the unnecessary parenthesis, and rather improve legibility with linebreaks?

Also we don't need to evaluate all that if is_admin() is false 🤔

```suggestion
		$is_in_app_marketplace = 'admin.php' === $pagenow
                         && isset( $_GET['page'] ) && 'wc-admin' === $_GET['page']
                         && isset( $_GET['path'] ) && '/extensions' === $_GET['path'];
```

Reply:
Nice, it's now visible that we have one redundant check for ` && isset( $_GET['page'] )` :)

Reply:
Nitpicks are always welcome, @wavvves

Updated it as per your suggestion, and moved that check so it's only done if `is_admin()` is true in 296f3f2885